### PR TITLE
Notes from Sept 22 learn team meeting

### PR DESCRIPTION
**Pull request by toddjordan**
Approvers:
@locks 
@acorncom 
@gavinjoyce
@wifelette 

Thumbsup this comment for approval

### CODE REVIEW

File: docs-team/2016-09/september-22.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+- DB: Will Combine various github repos into one place and put it on the web site.  Punch list for meetup organizers. 
+- DB: https://github.com/ember-learn/ember-help-wanted
+- DB: Tried firebase to host,  didn't work out - can't to querying very well, especially with permissions.  Tried couchdb, not a good fit.  Still looking for a backend technology.  Possibly Denali?  Tom Dale has been looking at it. Dave wasmer(?) is the contact on that.  Dave is willing to pay for backend hosting with the purpose of stress testing the technology.
+
+### Website
+
+- RM: Growing not to like the web site for various reasons.  Will ask the team for ideas on what do with it.  Intends to send Leah a proposal.
+- LS: Kris Seldon introduced me to someone wanting to redesign, but fell through.
+- RM: Get me your thoughts on what we need to do the web site
```
Comment (nathanhammond) [inline docs-team/2016-09/september-22.md]: New colors.

File: docs-team/2016-09/september-22.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+### More Learn Page
+
+- DB: would you (LS) agree that the learn page is blocked on styling?
+- RM: It could look better, but I think we are more blocked on advanced topics
+- DB: we should save advanced topics for later and expose it then.
+- RM: Lets resolve the menu bar discussion then.
+- LS: I think its intuitive that "Learn" would lead to guides and api, but I know that not everyone would agree.
+- RM: its what angular does
+- LS: could we just replace guides with Learn and leave api
+- DB: I would like to replace both guides and api with learn, but I know there would be pushback
```
Comment (nathanhammond) [inline docs-team/2016-09/september-22.md]: API is my most-pointed-to link when sitting at other peoples' desks, but I directly hit the URL at this point.

File: docs-team/2016-09/september-22.md (hunk 1 above)
Comment (locks) [inline docs-team/2016-09/september-22.md]: Nathan.

**Comment by acorncom** [comment]
Looks like we're good here ...

**Pull request merged**